cmd/config/init/init.go

```diff
@@ -20,16 +20,16 @@ func InitCmd() *cobra.Command {
 		Short: "Initialize a rollapp configuration on your local machine.",
 		Run: func(cmd *cobra.Command, args []string) {
 			initConfig := getInitConfig(cmd, args)
-			overwrite, err := prepareDirectory(initConfig.Home)
-			handleError(err)
+			overwrite, err := cleanHomeDir(initConfig.Home)
```

Comment:
for me `overwrite` returned from `cleanHomeDir` in unclear. also it seems like if the directory is clean, `overwrite` return `true` which IMO is counter intuitive.